packages/workflow/src/errors/node-api.error.ts

```diff
@@ -31,6 +31,7 @@ export interface NodeOperationErrorOptions {
 	messageMapping?: { [key: string]: string }; // allows to pass custom mapping for error messages scoped to a node
 	functionality?: Functionality;
 	type?: string;
+	allowRewrapping?: boolean;
```

Comment:
this seems to be falsy everywhere outside tests right now. can we remove this until we actually need it?